schemaregistry/spec.md

```diff
@@ -26,7 +26,7 @@ deserialize the event or message payload need access to the schema used to
 publish it.
 
 For formats that do not depend on external schemas for serialization, schemas
-might still be required in some scenarios to allow for consumers or inspecting
+might still be needed in some scenarios to allow for consumers or inspecting
```

Comment:
@clemensv can you check the RFC keywords in this file?

Reply:
If you need a review for those let's make that an issue.